test/unit/uk/gov/hmrc/tariffclassificationfrontend/controllers/CaseControllerSpec.scala

```diff
@@ -157,12 +165,35 @@ class CaseControllerSpec extends WordSpec with Matchers with WithFakeApplication
       given(eventService.getEvents(refEq(aCase.reference), refEq(NoPagination()))(any[HeaderCarrier])) willReturn successful(Paged.empty[Event])
       given(queueService.getAll) willReturn successful(Seq.empty)
 
-      val result = controller(aCase).addNote(aCase.reference)(aValidForm)
+      val result = controller(aCase, Set(Permission.ADD_NOTE)).addNote(aCase.reference)(aValidForm)
       status(result) shouldBe Status.OK
       contentType(result) shouldBe Some("text/html")
       charset(result) shouldBe Some("utf-8")
       contentAsString(result) should include("This field is required")
     }
+
+    "add note return expected view when has right permissions" in {
```

Comment:
This test doesn't actually assert on any expected view

It's also in a block called "Case Trader" which doesnt make sense

Reply:
Tidied the test name.  Its in the right block - testing the ability to add notes